access/pom.xml

```diff
@@ -252,5 +285,87 @@
             <artifactId>solr-search</artifactId>
             <version>${cdr.version}</version>
         </dependency>
+        <!-- integration testing -->
+        <dependency>
+            <groupId>org.fcrepo</groupId>
+            <artifactId>fcrepo-http-commons</artifactId>
+            <version>${fcrepo4.version}</version>
```

Comment:
wonder if we should think about changing the property namespace to be simply "fcrepo", since fcrepo5 is coming before too long? not that this change needs to happen on this PR